app/assets/javascripts/thyme/annotations/annotation.js

```diff
@@ -80,48 +80,9 @@ class Annotation {
     Returns a string with the correct translation of the category and subcategory of this annotation.
   */
   categoryLocale() {
-    let c, s;
-    switch (this.category) {
-      case "note":
-        c = document.getElementById('annotation-locales').dataset.note;
-        break;
-      case "content":
-        c = document.getElementById('annotation-locales').dataset.content;
-        break;
-      case "mistake":
-        c = document.getElementById('annotation-locales').dataset.mistake;
-        break;
-      case "presentation":
-        c = document.getElementById('annotation-locales').dataset.presentation;
-    }
-    if (this.subcategory === null) {
-      return c;
-    }
-    switch (this.subcategory) {
-      case "definition":
-        s = document.getElementById('annotation-locales').dataset.definition;
-        break;
-      case "argument":
-        s = document.getElementById('annotation-locales').dataset.argument;
-        break;
-      case "strategy":
-        s = document.getElementById('annotation-locales').dataset.strategy;
-    }
-    return c + " (" + s + ")";
-  }
-  
-  /* Returns a fixed color depending only on the category of the annotation. */
-  categoryColor() {
-    switch (this.category) {
-      case "note":
-        return "#44ee11"; //green
-      case "content":
-        return "#eeee00"; //yellow
-      case "mistake":
-        return "#ff0000"; //red
-      case "presentation":
-        return "#ff9933"; //orange
-    }
```

Comment:
Yeah 🎉, now this is what I call the benefits of encapsulation. Really cool approach with the "Enum" ;)